scipy/stats/stats.py

```diff
@@ -352,6 +352,10 @@ def hmean(a, axis=0, dtype=None):
         dtype of `a`, unless `a` has an integer `dtype` with a precision less
         than that of the default platform integer. In that case, the default
         platform integer is used.
+    allow_zero : bool, optional
+        If `allow_zero` is False then hmean will raise a ValueError if any of
+        the elements in a are 0. If `allow_zero` is True then hmean will return
+        0 if any of the elements in a are 0.
```

Comment:
Will probably want `versionadded` directive if we go with this. Also, perhaps some backticks around `a` as done elsewhere/above.

Reply:
> Will probably want versionadded directive if we go with this. Also, perhaps some backticks around a as done elsewhere/above.
Done.